bs/src/tablecloth.ml

```diff
@@ -191,7 +191,7 @@ module List = struct
     match l with [] -> None | _ :: rest -> Some rest
 
 
-  let append (l1 : 'a list) (l2 : 'a list) : 'a list = l1 @ l2
```

Comment:
because the belt version is optimized to be efficient when compiled to js
`
let concat xs ys =
  match xs with
  | [] -> ys
  | h::t ->
    let cell = mutableCell h [] in
    unsafeMutateTail (copyAuxCont t cell) ys;
    cell
`

Reply:
Interesting!